corehq/apps/reports/standard/monitoring.py

```diff
@@ -464,7 +464,7 @@ def rows(self):
             if self.all_relevant_forms:
                 for form in self.all_relevant_forms.values():
                     row.append(self._form_counts[
-                        (user.user_id, form['xmlns'], form['app_id'])
+                        (user.user_id, form['app_id'], form['xmlns'].lower())
```

Comment:
ah oops, read this as `row.append((user.user_id, form['app_id'], form['xmlns'].lower()))`. looks good thanks!